cmd/compose/publish.go

```diff
@@ -46,7 +46,7 @@ func publishCommand(p *ProjectOptions, dockerCli command.Cli, backend api.Servic
 		RunE: Adapt(func(ctx context.Context, args []string) error {
 			return runPublish(ctx, dockerCli, backend, opts, args[0])
 		}),
-		Args: cobra.ExactArgs(1),
+		Args: cli.RequiresMinArgs(1),
```

Comment:
why this change ? Don't we only support a single (required) arg for this command ?